### ISSUE TITLE
ppe42-binutils doesn't build with recent GCC

### ISSUE THREAD

**Issue by ghost** [open-power/ppe42-binutils#1]
https://github.com/open-power/op-build/issues/803

From that bug:

With GCC6:
```
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I. -I. -I./../include -DHAVE_bfd_elf32_powerpc_vec -DHAVE_rs6000coff_vec -DHAVE_bfd_elf32_powerpcle_vec -DHAVE_ppcboot_vec -DHAVE_bfd_elf64_powerpc_vec -DHAVE_bfd_elf64_powerpcle_vec -DHAVE_bfd_elf64_little_generic_vec -DHAVE_bfd_elf64_big_generic_vec -DHAVE_bfd_elf32_little_generic_vec -DHAVE_bfd_elf32_big_generic_vec -DBINDIR=\"/home/stewart/op-build/output/host/usr/powerpc64le-buildroot-linux-gnu/sysroot/ppe42-binutils/linux/bin\" -W -Wall -Wstrict-prototypes -Wmissing-prototypes -Wshadow -Werror -Wno-error=unused-value -MT elf64-ppc.lo -MD -MP -MF .deps/elf64-ppc.Tpo -c elf64-ppc.c  -fPIC -DPIC -o .libs/elf64-ppc.o
elf64-ppc.c: In function ‘ppc64_elf_size_stubs’:
elf64-ppc.c:12257:16: error: left shift of negative value [-Werror=shift-negative-value]
          & (-1 << htab->plt_stub_align));
                ^~
elf64-ppc.c: In function ‘ppc64_elf_build_stubs’:
elf64-ppc.c:12725:12: error: left shift of negative value [-Werror=shift-negative-value]
      & (-1 << htab->plt_stub_align));
            ^~
cc1: all warnings being treated as errors
Makefile:1612: recipe for target 'elf64-ppc.lo' failed
make[6]: *** [elf64-ppc.lo] Error 1
Makefile:1654: recipe for target 'all-recursive' failed
make[5]: *** [all-recursive] Error 1
Makefile:1138: recipe for target 'all' failed
make[4]: *** [all] Error 2
Makefile:2598: recipe for target 'all-bfd' failed
make[3]: *** [all-bfd] Error 2
Makefile:829: recipe for target 'all' failed
make[2]: *** [all] Error 2
package/pkg-generic.mk:216: recipe for target '/home/stewart/op-build/output/build/host-ppe42-binutils-5b161fc30519a965f16e7e73c3410a388140cba1/.stamp_built' failed
```

**Comment by ghost** [comment]
From op-build issue: Commit 29f628db8 from upstream binutils needs to be backported. Or, more sensibly, ppe42 needs to make its way upstream.

**Comment by dgilbert999** [comment]
Resolved with 394161cb79956f4297fea1638ec4c42afeba77c9
